src/components/Card/Card.component.tsx

```
@@ -14,21 +21,35 @@ const Card = ({
     const theme = useTheme();
 
     return (
-        <StyledCardContainer onClick={handleClick}>
+        <StyledCardContainer
+            tabIndex={0}
+            role="link"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<a>
<link>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)